freqtrade/rpc/telegram.py

```diff
@@ -213,7 +226,7 @@ def _daily(self, bot: Bot, update: Update) -> None:
         :return: None
         """
         stake_cur = self._config['stake_currency']
-        fiat_disp_cur = self._config['fiat_display_currency']
+        fiat_disp_cur = self._config.get('fiat_display_currency', None)
```

Comment:
is there a reason not to default to `""` instead of `None` seeing as we mostly just `f'Profit {fiat_disp_cur if fiat_disp_cur else ""}'`?